### ISSUE TITLE
Mask not working

### ISSUE THREAD

**Issue by wwx13** [lucidrains/h-transformer-1d#15]
def forward(self, x, mask = None):
        b, n, device = *x.shape, x.device
        assert n <= self.max_seq_len, 'sequence length must be less than the maximum sequence length'
        x = self.token_emb(x)
        x = self.layers(x)
        return self.to_logits(x)

I think...
Masking  does not work ???

**Comment by wwx13** [comment]
Hey, I wonder if H-transformer support using input mask ?
Hope your reply ……^ ^

**Comment by lucidrains** [comment]
@wwx13 oops! thanks for catching this!
